content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -24,9 +24,7 @@ const CALENDAR_HEADINGS = {
 
 const ActivityFeedItem = React.createClass({
   getInitialState() {
-    return {
-      showContextMenu: false
-    };
+    return {showContextMenu: false};
```

Comment:
This was autofixed with http://eslint.org/docs/rules/object-curly-newline where newlines inside curlies are used only when the object has multiple lines.